mentoring/tests/integration/test_data_export.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
```

Comment:
This feature was removed due to performance issues - we should remove it from here too.